src/common.c

```diff
@@ -555,6 +555,35 @@ int ofi_addr_cmp(const struct fi_provider *prov, const struct sockaddr *sa1,
 	}
 }
 
+int ofi_is_only_port_set(const char *node, const char *service,
```

Comment:
I would rename to only_src_port_set, since the function returns 0 if a dest_addr is provided

Reply:
Sure. Also I should add a check for flags since node and service would indicate source only when FI_SOURCE is set.